pages/linux/modinfo.md

```diff
@@ -0,0 +1,7 @@
+# modinfo
+
+> Extract information about a Linux Kernel module.
```

Comment:
linux kernel. Lower case.

Reply:
Made kernel lowercase. I think Linux should stay as it is.